install_gcc.md

```diff
@@ -0,0 +1,50 @@
+## Installing gcc and binutils for `arm-none-eabi`.
+
+`c-of-time` and `Rust of Darkness` require a GCC & Binutils toolchain for `arm-none-eabi`.
+You have several ways of installing it.
+
+The following guides for each platform are sorted by what's recommended (based on what we think might be easiest). 
+Follow any ONE of these guides.
+
+After reading the guide for your platform, make sure to read the section `Updating PATH`,
+since the tools need to be in your platform's `PATH`.
+
+### Windows
+- Download it from ARM, see `All Platforms` below.
+- Download it as part of devkitpro, see `All Platforms` below.
+- Install it using the Windows Subsystem for Linux and following the `Linux` instructions below.
+- Install it via Msys2's Mingw environment:
+   1. Install [Msys2](https://www.msys2.org/wiki/MSYS2-installation/)
+   2. Inside the MingW shell for your architecture (32bit or 64bit) install the package 
+      [mingw-w64-arm-none-eabi-gcc](https://packages.msys2.org/base/mingw-w64-arm-none-eabi-gcc) for your MingW version.
+
+### macOS
+- Install it via brew: [gcc-arm-embedded cask](https://formulae.brew.sh/cask/gcc-arm-embedded).
+- Download it from ARM, see `All Platforms` below.
+- Download it as part of devkitpro, see `All Platforms` below.
+
+### Linux
+- Install it via your system's package manager:
+  - Debian/Ubuntu: `sudo apt install gcc-arm-none-eabi binutils-arm-none-eabi`
+  - Arch Linux: `sudo pacman -Syu arm-none-eabi-gcc arm-none-eabi-binutils`
+  - Fedora: `sudo dnf install arm-none-eabi-binutils-cs arm-none-eabi-gcc-cs`
+- Download it from ARM, see `All Platforms` below.
+- Download it as part of devkitpro, see `All Platforms` below.
+
+### All platforms
+- [Download it from ARM](https://developer.arm.com/downloads/-/gnu-rm)
```

Comment:
```suggestion
- [Download it from ARM](https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads)
```
Sorry, I sent you a link to an outdated page with versions that are no longer updated in DMs. This link seems to be the correct one.